tests/ui/needless_for_each_fixable.fixed

```diff
@@ -110,7 +110,7 @@ fn should_not_lint() {
         }),
     }
 
-    // `for_each` is in a let bingind.
+    // `for_each` is in a let bingint.
```

Comment:
Should be 'binding'.

Reply:
Sigh. I somehow read it as a typo of bigint. I totally missed that the `n` there hinted at it being a letter swap.